<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoystickOptionsWidget</class>
 <widget class="QWidget" name="JoystickOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Axis</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Buttons</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="versionLabel">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="axisLabel">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="buttonsLabel">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QGroupBox" name="mappingBox">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Mapping</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="infoPushButton">
        <property name="text">
         <string>Info</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/main.qrc">
          <normaloff>:/new/prefix1/info_rhombus.png</normaloff>:/new/prefix1/info_rhombus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openPushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/main.qrc">
          <normaloff>:/new/prefix1/play.png</normaloff>:/new/prefix1/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/main.qrc">
          <normaloff>:/new/prefix1/stop_2.png</normaloff>:/new/prefix1/stop_2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recordPushButton">
        <property name="text">
         <string>Record</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/main.qrc">
          <normaloff>:/new/prefix1/record.png</normaloff>:/new/prefix1/record.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="GraphPlotDialog" name="graphWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deviceLineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="rereadButton">
       <property name="toolTip">
        <string>Refresh</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/main.qrc">
         <normaloff>:/new/prefix1/refresh.png</normaloff>:/new/prefix1/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphPlotDialog</class>
   <extends>QWidget</extends>
   <header>uis/graphPlotDialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../resources/main.qrc"/>
  <include location="../../../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>deviceLineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rereadButton</sender>
   <signal>released()</signal>
   <receiver>JoystickOptionsWidget</receiver>
   <slot>rereadDevices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infoPushButton</sender>
   <signal>released()</signal>
   <receiver>JoystickOptionsWidget</receiver>
   <slot>getProps()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openPushButton</sender>
   <signal>released()</signal>
   <receiver>JoystickOptionsWidget</receiver>
   <slot>openJoystick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closePushButton</sender>
   <signal>released()</signal>
   <receiver>JoystickOptionsWidget</receiver>
   <slot>closeJoystick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordPushButton</sender>
   <signal>released()</signal>
   <receiver>JoystickOptionsWidget</receiver>
   <slot>recordJoystick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rereadDevices()</slot>
  <slot>getProps()</slot>
  <slot>openJoystick()</slot>
  <slot>closeJoystick()</slot>
  <slot>recordJoystick()</slot>
 </slots>
</ui>
